setup.py

```diff
@@ -100,6 +100,7 @@
     "tifffile": ["tifffile"],
     "pyav": ["av"],
     "heif": ["pillow-heif"],
+    "avif": ["pillow-heif"],
```

Comment:
Actually, is `avif` supported by the same package that supports `heif`? 

If so, we should be able to import both at the same time right? Since either both are present or none.

Reply:
Yes indeed. I may make a commit.